.asf.yaml

```diff
@@ -33,6 +33,14 @@ github:
     - database
   protected_branches:
     main: {}
+  environments:
+    databricks-e2e:
+      wait_timer: 0
+      required_reviewers:
+        - alexguo-db
+        - jadewang-db
+      deployment_branch_policy:
+        protected_branches: true
```

Comment:
should probably add these to a `collaborators` key https://github.com/apache/infrastructure-asfyaml/blob/main/README.md#assigning-the-github-triage-role-to-external-collaborators